corehq/apps/analytics/templates/analytics/fullstory.html

```diff
@@ -1,8 +1,8 @@
-{% if FULLSTORY_ID and not request.project.restrict_superusers %}
+{% if ANALYTICS_IDS.FULLSTORY_ID and not request.project.restrict_superusers %}
```

Comment:
@orangejenny you probably missed this because it's not in the locasettings

Reply:
So I did - thank you for fixing, and also for adding to localsettings.